issue_router/models.py

```diff
@@ -4,7 +4,10 @@
 
 class CreateIssueRequest(BaseModel):
     """Schema for a /create_issue request."""
+    title: str
     url: str
+    submitter: str
```

Comment:
One thing to note is that I think this field should be optional. There was discussion when creating this tool about including this field and I believe the conclusion was that feedback submitters should be able to stay anonymous.

Reply:
agree